Contour.appdata.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="runtime">
```

Comment:
this is NOT a runtime.

Reply:
I was apparently looking at the wrong example. I now found https://freedesktop.org/software/appstream/docs/sect-Metadata-Application.html and will adapt that tonight. Thanks.